packages/components/organisms/f-checkout/src/services/addressService.js

```diff
@@ -0,0 +1,85 @@
+import { getCookie } from '../utils/helpers';
```

Comment:
I'd be tempted to call this file `address.service.js`. Not a blocker for merging but maybe something to consider as a  naming conventions for service files like this. Example: [searchbox](https://github.com/justeat/fozzie-components/tree/master/packages/components/molecules/f-searchbox/src/services).